site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -1,152 +1,137 @@
 # Cross-Namespace routing
 
 The Gateway API has core support for cross Namespace routing. This is useful
-when more than one user or team is sharing the underlying networking infrastructure,
-yet control and configuration must be segmented to minimize access and fault
-domains. Gateways and Routes can be deployed into different Namespaces and bind
-with each other across Namespace boundaries. This allows differing user access
-and roles (RBAC) to be applied to separate Namespaces, effectively controlling
-who has access to different parts of the cluster-wide routing configuration. The
-ability for Routes to bind with Gateways across Namespace boundaries is goverend
-by [_Route binding_](#cross-namespace-route-binding), which
-is explored in this guide. The guide shows how two independent teams can safely
-share the same Gateway from different Namespaces.
+when more than one user or team is sharing the underlying networking
+infrastructure, yet control and configuration must be segmented to minimize
+access and fault domains.
+
+Gateways and Routes can be deployed into different Namespaces and attached to
+each other across Namespace boundaries. This allows differing user access and
```

Comment:
```suggestion
Gateways and Routes can be deployed into different Namespaces and Routes 
attached to Gateways across Namespace boundaries. This allows differing user access and
```

I go into this further down the review, but I think that the way we need to discuss this is that Routes want to be/request to be attached, and Gateways allow the attachment.